.github/workflows/build.yml

```diff
@@ -19,6 +19,7 @@ jobs:
       # When Chrome version is specified, we pin to a specific version of Chrome
       # Comment this out to use the latest release
       #CHROME_VERSION: "90.0.4430.212-1"
+      NODE_OPTIONS: "--max_old_space_size=4096"
```

Comment:
I think this setting is (strangely) causing issues with CodeCov checks.  See this "known issue" with the Codecov uploader (which we use): https://github.com/codecov/uploader/issues/475

I dug into this because I noticed that Codecov is consistently failing to update for this PR (even after I reran it several times).  While Codecov is "flakey" at times, usually it works if you simply rerun it. It's only this PR that seems to consistently fail to update.

Per that ticket above, the only workarounds so far are to either:
1.  Remove `NODE_OPTIONS`  (which might be an option if this is unnecessary)
2. Or, **unset** `NODE_OPTIONS`  just before CodeCov runs, i.e. just before `codecov/codecov-action@v2` runs (as this is what is currently failing).

Reply:
We'd put it there to see if it would solve with the SSR issue ↑ (it seemed to help a little, but not consistentlty). The underlying issue was fixed with e877f96d, we don't need to set `NODE_OPTIONS`.